dora/core/common/src/main/java/alluxio/exception/runtime/AlluxioRuntimeException.java

```diff
@@ -112,6 +113,17 @@ public StatusRuntimeException toGrpcStatusRuntimeException() {
         .asRuntimeException(getMetadata());
   }
 
+  /**
+   * Wrap a RejectedExecutionException with io.grpc RESOURCE_EXHAUSTED
+   * exception.
+   * @param ex
+   * @return AlluxioRuntimeException
+   */
+  public static AlluxioRuntimeException from(RejectedExecutionException ex) {
+    return new AlluxioRuntimeException(Status.RESOURCE_EXHAUSTED, "StageOverload",
+        ex, ErrorType.Internal, true);
```

Comment:
this is probably a user error. Internal usually means code bug

Reply:
done